huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go

```diff
@@ -128,6 +128,12 @@ func ResourceDcsInstance() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
+				DiffSuppressFunc: func(k, old string, new string, d *schema.ResourceData) bool {
```

Comment:
thanks for the contribution, it's better to use `DiffSuppressFunc: utils.SuppressCaseDiffs,`